packit_service/models.py

```diff
@@ -4648,3 +4650,8 @@ def get_onboarded_projects() -> tuple[dict[int, str], dict[int, str]]:
         for project in recheck_if_onboarded.difference(onboarded_projects)
     }
     return (onboarded, almost_onboarded)
+
+
+class BodhiUpdatesListResponse(BaseModel):
+    result: List
+    status: HTTPStatus
```

Comment:
as this file representes DB models, this model should be rather placed in the appropriate API module with FastAPI implementation. 

The structure also doesn't look correct, please refer to the current dictionary that is being returned, you can also try it out e.g. using [staging Swagger UI](https://stg.packit.dev/api/)